js/alerts.js

```diff
@@ -196,6 +197,146 @@ function updateStatus(alertid, duplicate, bugid, mergedfrom) {
+}
+
+function performAction(){
+    var action = $(document.getElementById("actions")).val();
+    console.log(action);
+
+    var status_options = ["NEW", "Back Filling", "Investigating", "Duplicate", "Resolved", "Shipped", "Wont Fix", "False Alarm", "Ignore", "Not Tracking", "Backout", "Too Low"];
+
+    //get ids of all the checked alerts
+   var checkedIds = $(":checkbox:checked").map(function() {
+        return this.id;
+    }).get();
+   //Check if atleast one alert is chosen
+   if (checkedIds.length<=0 && action != "Actions"){
+    console.log("No alert chosen");
+    alert("Please Choose atleast one Alert");
+   }
+   else{
+
+        for (alert in checkedIds){
+            checkedIds[alert] = checkedIds[alert].split('-')[1];
+        }
+        console.log(checkedIds);
+        //Check if status has to be changed
+        if (containsObject(action, status_options)){
+            console.log("change status-"+action);
+            //Update the status one by one by calling updatestatus()
+            for (id in checkedIds){
+                $.ajax({
+                     url: root_url + "/updatestatus",
+                    type: "POST",
+                    data: {
+                        id: checkedIds[id],
+                        status: action,
+                    }
+                });
+            }
+            location.reload();
+        }
+
+        else if(action == "Change Revision"){
+            console.log("Reached changing revision");
+            var newRev = prompt("Please enter new Revision");
+            for (id in checkedIds){
+
+                var bug = '1234';
+                if (bug == '') {
+                    bug = $(document.getElementById(alertid + "-bug")).val();
+                }
+    
+                if (newRev != null) {
+                    console.log('new rev-'+newRev);
+                    $.ajax({
+                         url: root_url + "/updaterev",
+                        type: "POST",
+                        data: {
+                            id: checkedIds[id],
+                            revision: newRev,
+                        }
+                    });
+                }
+            }
+            location.reload();
+
+        }
+
+        else if(action == "Add Bug"){
+            console.log("REACHED ADD BUG")
+            var BugID = prompt("Please enter Bug ID");
+            console.log('BUG ID-'+BugID);
+            if (BugID != null){
+                for(id in checkedIds){
+                    $.ajax({
+                         url: root_url + "/updatefields?type=bug",
+                        type: "POST",
+                        data: {
+                            id: checkedIds[id],
+                            BugID: BugID,
+                        }
+                    });
+
+                }
+                location.reload();
+            
+            }
+        }
+
+        else if(action == "Add Comment"){
+            console.log("REACHED Add Comment")
```

Comment:
yes!  just the console.log line

On Wed, Jan 21, 2015 at 11:58 AM, Vikas Mishra <notifications@github.com>
wrote:

> In js/alerts.js
> <https://github.com/jmaher/alert_manager/pull/126#discussion_r23314557>:
>
> > +                         url: root_url + "/updatefields?type=bug",
> > +                        type: "POST",
> > +                        data: {
> > +                            id: checkedIds[id],
> > +                            BugID: BugID,
> > +                        }
> > +                    });
> > +
> > +                }
> > +                location.reload();
> > +
> > +            }
> > +        }
> > +
> > +        else if(action == "Add Comment"){
> > +            console.log("REACHED Add Comment")
>
> you're talking about the console.log() line?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jmaher/alert_manager/pull/126/files#r23314557>.
>